### ISSUE TITLE
Multiple Configs/processes

### ISSUE THREAD

**Issue by captcho** [modrzew/pokeminer#226]
I want to run 2x processes of worker.py and web.py (on different ports). Is it possible?

**Comment by Kobito123** [comment]
i did this for the last couple of weeks but now my bots sometimes dont work. atleast they dont set icons on the map. donno if this has anything to do with it

**Comment by Aiyubi** [comment]
yes of course. Just clone the project into 2 different directorys. Really no problem (aside from possible IP bans). You can even decide if you want to use the same database or 2 different ones (from the sound of it you want different ones)
